test/include.test.js

```diff
@@ -40,7 +41,8 @@ describe('include', function() {
     });
   });
 
-  it('should fetch hasMany relation', function(done) {
+  bdd.itIf(connectorCapabilities.supportNonPrimaryKeyIN !== false,
```

Comment:
For the full fledged lusion implementation to allow `=` , an issue has been filed as:
https://waffle.io/strongloop-internal/scrum-asteroid/cards/59135f6e0fe1c5009245818f